common/formatting/line_wrap_searcher_test.cc

```diff
@@ -373,28 +373,28 @@ TEST_F(SearchLineWrapsTestFixture, FitsOnLine) {
   ftokens_in[0].before.spaces_required = 99;  // irrelevant
   ftokens_in[1].before.spaces_required = 1;
   ftokens_in[2].before.spaces_required = 1;
-  EXPECT_TRUE(FitsOnLine(uwline_in, style_));
+  EXPECT_TRUE(FitsOnLine(uwline_in, style_).fits);
```

Comment:
Also test `EXPECT_EQ` for `.final_column`, can just take the expected value from the comments.

Reply:
Done